### DISCUSSION TITLE
Mode to detect spam only from verified channels

### DISCUSSION THREAD

**Discussion by KendallDoesCoding** [ThioJoe/YT-Spammer-Purge#396] [Ideas]
There's something going around on youtube where people sell verified accounts to people and they comment on every video, copying the same comment of a non-verified person because people will then click on the verified commenter thinking its the original comment to check their channel out and there there's a video called "Who is {channelname}" where I think they tell people also how to buy verified accounts, because verified accounts can give a bad influence to people I think a basic mode to detect verified commenters who have copied a non-verified persons comment would be good.

Sorry this is complicated, I can explain further incase you didnt understand @ThioJoe

**Comment by TechStudent10** [comment]
Not sure if there is a way to differentiate non-verified from verified in the YouTube API.
